src/service/token.service.js

```diff
@@ -0,0 +1,29 @@
+import { Token } from '../modules/Token.js';
+
+async function save(userId, newToken) {
+  const token = await Token.findOne({ where: { userId } });
+
+  if (!token) {
+    await Token.create({ userId, refreshToken: newToken });
+
+    return;
+  }
+
+  token.refreshToken = newToken;
+
+  await token.save();
+}
+
+function getByToken(refreshToken) {
```

Comment:
Consider adding error handling to manage cases where the token retrieval might fail due to database issues.